contracts/FungibleToken.cdc

```diff
@@ -193,6 +195,24 @@ pub contract interface FungibleToken {
                     "New Vault balance must be the sum of the previous balance and the deposited Vault"
             }
         }
+
+        /// Function that returns all the Metadata Views implemented by a Fungible Token
+        ///
+        /// @return An array of Types defining the implemented views. This value will be used by
+        ///         developers to know which parameter to pass to the resolveView() method.
+        ///
+        pub fun getViews(): [Type]{
```

Comment:
@joshuahannan I've done the default implementations of the Resolver functions on this branch since it seemed to me that it belonged more to the features than to the example, does it look good?

Reply:
yeah, that looks good. The version of Cadence that you're using to test it supports default implementations and is working, right?

Reply:
Not at all, I'm on v0.39.3 of the cli, the latest distributed via brew. I thought the version supporting default implementations was not live yet. I haven't merge this changes into the example token branch which is the one that includes the tests.

<img width="1920" alt="Screenshot 2022-09-27 at 14 17 43 (2)" src="https://user-images.githubusercontent.com/6607145/192524447-ebc04f1f-0ab3-42b9-b52c-46bd20e9f69e.png">

Reply:
Working with v0.41.0 🎉 
<img width="903" alt="Screenshot 2022-10-06 at 18 49 37" src="https://user-images.githubusercontent.com/6607145/194372265-4db040f7-ed2f-4e78-8f33-959b545e11d1.png">